### ISSUE TITLE
New update does not build anymore

### ISSUE THREAD

**Issue by elabz** [fboulnois/llama-cpp-docker#11]
Hello there, thanks for the update yesterday!
Obviously, it requires a rebuild, which is now failing:

```
 => [env-build 4/4] RUN git clone https://github.com/ggerganov/llama.cpp.git   && cd llama.cpp   && make -j LLAMA_CUDA=1 CUDA_DOCKER_ARCH=all                                                                 1489.3s
 => CACHED [env-deploy  3/11] COPY --from=0 /usr/lib/x86_64-linux-gnu/libgomp.so.1 /usr/local/lib/libgomp.so.1                                                                                                   0.0s
 => CACHED [env-deploy  4/11] COPY --from=0 /usr/local/cuda/lib64/libcublas.so.12 /usr/local/lib/libcublas.so.12                                                                                                 0.0s
 => CACHED [env-deploy  5/11] COPY --from=0 /usr/local/cuda/lib64/libcublasLt.so.12 /usr/local/lib/libcublasLt.so.12                                                                                             0.0s
 => CACHED [env-deploy  6/11] COPY --from=0 /usr/local/cuda/lib64/libcudart.so.12 /usr/local/lib/libcudart.so.12                                                                                                 0.0s
 => ERROR [env-deploy  7/11] COPY --from=0 /srv/llama.cpp/main /usr/local/bin/llama                                                                                                                              0.0s
 => ERROR [env-deploy  8/11] COPY --from=0 /srv/llama.cpp/server /usr/local/bin/llama-server                                                                                                                     0.0s
------
 > [env-deploy  7/11] COPY --from=0 /srv/llama.cpp/main /usr/local/bin/llama:
------
------
 > [env-deploy  8/11] COPY --from=0 /srv/llama.cpp/server /usr/local/bin/llama-server:
------
Dockerfile:24
--------------------
  22 |     # copy llama.cpp binaries
  23 |     COPY --from=0 /srv/llama.cpp/main /usr/local/bin/llama
  24 | >>> COPY --from=0 /srv/llama.cpp/server /usr/local/bin/llama-server
  25 |
  26 |     # create llama user and set home directory
--------------------
ERROR: failed to solve: failed to compute cache key: failed to calculate checksum of ref 8fe31955-4f2d-44a4-b31c-bbddec1296a5::st3s3k9j3pyps58urond57mzj: "/srv/llama.cpp/server": not found

```

So, apparently `make` names the file differently now, not `main` or the location is somewhere that's not in the root dir of the repo? 🤔

**Comment by elabz** [comment]
Ah, yes! Oh, wow, that was unexpected. Merged pull request in `llama.cpp` from yesterday has this:
`build: rename main → llama-cli, server → llama-server, llava-cli → …` 

So, not `main`, `llama-cli`

**Comment by elabz** [comment]
```
COPY --from=0 /srv/llama.cpp/llama-cli /usr/local/bin/llama
COPY --from=0 /srv/llama.cpp/llama-server /usr/local/bin/llama-server
```

**Comment by fboulnois** [comment]
Fixed in v1.7.1.
